Modules/atexitmodule.c

```diff
@@ -192,31 +217,39 @@ atexit_register(PyObject *module, PyObject *args, PyObject *kwargs)
     }
 
     struct atexit_state *state = get_atexit_state();
+    /* In theory, we could hold the lock for a shorter amount of time
+     * using some fancy compare-exchanges with the length. However, I'm lazy.
```

Comment:
If you use "I" (I'm lazy) in the comment, I suggest to add your name as a prefix of the comment.